### ISSUE TITLE
Add smoke to parts

### ISSUE THREAD

**Issue by Aree-Vanier** [Aree-Vanier/Focus-Program-Culminating#28]
- [x] Add smoke particle effect to all gameObjects with part script
- [ ] Smoke triggers when the part.detach() is called
